src/Uno.UI/UI/Xaml/FrameworkElement.Layout.wasm.cs

```diff
@@ -149,13 +151,12 @@ private void InnerArrangeCore(Rect finalRect)
 
 			if (allowClipToSlot && !needsClipToSlot)
 			{
-				if (arrangeSize.Width < _unclippedDesiredSize.Width - SIZE_EPSILON)
+				if (IsLessThanAndNotCloseTo(arrangeSize.Width, _unclippedDesiredSize.Width))
 				{
 					_logDebug?.Debug($"{DepthIndentation}{this}: (arrangeSize.Width) {arrangeSize.Width} < {_unclippedDesiredSize.Width}: NEEDS CLIPPING.");
 					needsClipToSlot = true;
 				}
-
-				if (arrangeSize.Height < _unclippedDesiredSize.Height - SIZE_EPSILON)
+				else if (IsLessThanAndNotCloseTo(arrangeSize.Height, _unclippedDesiredSize.Height))
```

Comment:
Not sure if it's expected, but you changed this the opposite way in the _Layouter.cs_ (removed the `else`)

Reply:
You're comparing phase 1 with phase 2...  there's no disrutpancies :-)